docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -13,15 +13,15 @@ labels:
     - enterprise
     - oss
 menuTitle: Azure AD OAuth2
-title: Configure Azure AD OAuth2 authentication
+title: Configure Azure AD/Entra ID OAuth2 authentication
 weight: 800
 ---
 
-# Configure Azure AD OAuth2 authentication
+# Configure Azure AD/Entra ID OAuth2 authentication
 
 The Azure AD authentication allows you to use an Azure Active Directory tenant as an identity provider for Grafana. You can use Azure AD application roles to assign users and groups to Grafana roles from the Azure Portal.
```

Comment:
I'd call out here that Azure AD is not called Microsoft Entra ID

Reply:
https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id says that Azure AD is Microsoft Entra ID, but we'll update these in a follow up PR.